build.gradle

```diff
@@ -381,10 +381,10 @@ task buildRca() {
         exec {
             workingDir("$rcaProjectDir")
             if (buildVersionQualifier == null || buildVersionQualifier == '' || buildVersionQualifier == 'null') {
-                commandLine './gradlew', 'build', '-x', 'test', "-Dopensearch.version=${opensearch_version}", "-Dbuild.snapshot=${isSnapshot}"
+                commandLine './gradlew', 'build', '-x', 'test', '-x', 'jacocoTestCoverageVerification', "-Dopensearch.version=${opensearch_version}", "-Dbuild.snapshot=${isSnapshot}"
```

Comment:
I guess added my mistake?

Reply:
As tests are excluded, the ```jacocoTestCoverageVerification``` task should also be ignored.